src/components/atoms/dropdown/modal.tsx

```diff
@@ -67,10 +67,7 @@ export const DropdownModal: React.FC<DropdownModalProps> = ({
   useEffect(() => {
     if (search && !screenReaderEnabled) {
       const focusInput = () => searchInputRef.current?.focus();
-
-      // On Android, bringing up the keyboard during render causes juddering
-      // and can cause sizes to end up incorrectly calculated
-      Platform.OS === 'android' ? setTimeout(focusInput, 400) : focusInput();
+      focusInput();
```

Comment:
Isn't "bringing up the keyboard during render causes juddering	and can cause sizes to end up incorrectly calculated" happening on Android anymore? 😂

Reply:
That's what the change from `height` to `padding` does - the root cause of the juddering was the content height changing during render.

However... while I was testing https://github.com/covid-alert-ny/covid-green-app/pull/580/commits/2e8617446f4030aed9a2fa59815fc5a400554183 to confirm we did still need this for older Android I found a horrible bug where the padding was effectively applied twice after the first render. 

I've reverted all the nice-to-have changes here and this PR just fixes the search issue.